src/network/networkprotocol.h

```diff
@@ -612,6 +612,13 @@ enum ToClientCommand
 
 	TOCLIENT_CLOUD_PARAMS = 0x54,
 	/*
+	 	u16 id_player_knockback
+	 	v3f direction
+	 	f32 time_knockback
+	*/
+	TOCLIENT_KNOCK_BACK = 0x55,
```

Comment:
The packet param description is always defined AFTER the packet itself, thus move this part below
```
	f1000 thickness
	v2f1000 speed
*/
```